services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet, Q
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    query = Q()
+
+    if genres_ids is not None:
+        query &= Q(genres__id__in=genres_ids)
+
+    if actors_ids is not None:
+        query &= Q(actors__id__in=actors_ids)
+
+    return Movie.objects.filter(query)
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None,
+) -> None:
+    movie = Movie.objects.create(
+        title=movie_title,
+        description=movie_description,
+    )
+    if genres_ids is not None:
```

Comment:
Can we try to refactor this place and all other places in PR.
`if genres_ids is not None:` do we really need this extra check:  `is not None`

Reply:
Yes. Explicit is better than implicit - Zen of Python